pom.xml

```diff
@@ -9,13 +9,22 @@
     <packaging>pom</packaging>
     <version>1.0-SNAPSHOT</version>
     <modules>
-      <module>DAL</module>
-      <module>core</module>
-      <module>JobRegistration</module>
-      <module>service</module>
+        <module>JobRegistration</module>
+        <module>service</module>
+        <module>models</module>
+        <module>DAL</module>
+        <module>core</module>
+        <module>JobRegistration</module>
```

Comment:
JobRegistration is mentioned twice in this pom.

Reply:
removed